pyproject.toml

```
@@ -1,3 +1,109 @@
 [build-system]
 requires = ["setuptools >= 66.1.0"]  # minimum setuptools version supporting Python 3.12
 build-backend = "setuptools.build_meta"
+
+[project]
+requires-python = ">=3.11"
+name = "ansible-core"
+authors = [
+    {name = "Ansible Project"},
+]
+description = "Radically simple IT automation"
+readme = "README.md"
+license = {text = "GPLv3+"}
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Information Technology",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+    "Natural Language :: English",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: System :: Installation/Setup",
+    "Topic :: System :: Systems Administration",
+    "Topic :: Utilities",
+]
+dynamic = ["version", "dependencies"]
+
+[project.urls]
+"Homepage" = "https://ansible.com/"
+"Source Code" = "https://github.com/ansible/ansible/"
+"Bug Tracker" = "https://github.com/ansible/ansible/issues/"
+"CI: Azure Pipelines" = "https://dev.azure.com/ansible/ansible/"
+"Documentation" = "https://docs.ansible.com/ansible-core/"
+"Code of Conduct" = "https://docs.ansible.com/ansible/latest/community/code_of_conduct.html"
+
+[tool.setuptools.dynamic]
+version = {attr = "ansible.release.__version__"}
+dependencies = {file = "requirements.txt"}
+
+[tool.setuptools]
+include-package-data = false
```

This ensures that all license files are included in the wheel

```suggestion
include-package-data = false
license-files = [
    "COPYING",
    "licenses/*.txt",
]
```